src/api/exporters/ezsLocals/linkDataset.js

```diff
@@ -1,18 +1,52 @@
+import validUrl from 'valid-url';
+import { set } from 'lodash';
+import deepObject from 'deep-object-map';
+
+/**
+ * replace uri blankNode to unique uri
+ * @param {object} data
+ */
+function blankNodeSpecifier(data) {
```

Comment:
It's weird for a function to have a non-verbal name (`blankNodeSpecify`, for example).